core/src/main/scala/com/pingcap/tispark/write/TiBatchWriteTable.scala

```diff
@@ -357,6 +357,15 @@ class TiBatchWriteTable(
     }
   }
 
+  def checkAuthorization(tiAuthorization: Option[TiAuthorization], options: TiDBOptions): Unit = {
+    if (options.replace) {
```

Comment:
Can you check if the replace has the same semantics as MySQL replace.
mysql replace need insert and delete pri https://dev.mysql.com/doc/refman/8.0/en/replace.html